SmartDeviceLink/SDLProxy.m

```diff
@@ -447,6 +444,10 @@ - (void)handleRPCDictionary:(NSDictionary<NSString *, id> *)dict {
     }
 
     [self sdl_invokeDelegateMethodsWithFunction:functionName message:newMessage];
+    
+    if ([functionName isEqualToString:SDLRPCFunctionNameOnAppInterfaceUnregistered] || [functionName isEqualToString:SDLRPCFunctionNameUnregisterAppInterface]) {
```

Comment:
Add a comment that interception of `SDLRPCFunctionNameOnAppInterfaceUnregistered` must be done **after* * it is broadcast as a notification to prevent reconnection attempts in the lifecycle manager when the `AppInterfaceUnregisteredReason` should prevent reconnections.